docs/third_party.rst

```diff
@@ -207,7 +208,7 @@ period ask to raise the status.
   An aiohttp middleware for reporting errors to Sentry. Python 3.5+ is required.
 
 - `aiohttp-datadog <https://github.com/underyx/aiohttp-datadog>`_
-  An aiohttp middleware for reporting metrics to Datadog. Python 3.5+ is required.
+  An aiohttp middleware for reporting metrics to DataDog. Python 3.5+ is required.
```

Comment:
It's capitalized as `Datadog`: https://www.datadoghq.com/about/press/

Reply:
I don't care too much.
If you like to keep a capitalized version -- please add `Datadog` to spell checker white list `./docs/spelling_wordlist.txt`

Reply:
Nah, I don't mind, I just thought you did 🙃